NARFO_BE/NARFO_BE/Controllers/ActionsController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using NARFO_BE.Models;
+
+namespace NARFO_BE.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ActionsController : ControllerBase
+    {
+        private readonly narfoContext _context;
+
+        public ActionsController(narfoContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Actions
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Models.Action>>> GetAction()
+        {
+            return await _context.Action.ToListAsync();
+        }
+
+        // GET: api/Actions/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Models.Action>> GetAction(int id)
+        {
+            var action = await _context.Action.FindAsync(id);
+
+            if (action == null)
+            return NotFound();
```

Comment:
Please format all files. Indentation is still important

Reply:
Also be choose a standard for your braces { }

Some if statements have them. Others do not have any like above

Reply:
![image](https://user-images.githubusercontent.com/46738448/52552254-39fc2800-2de8-11e9-964d-4e5fd0b28740.png)


you guys seem to be contradicting yourselves.....

Reply:
Rashads comment was for knowledge purpose, he did not require you to change. However you have to stick to one standard. Cannot have both in your code.

Reply:
and we made that indentation error trying to solve the curl braces problem

Reply:
What was the curly brace problem again? 
You can use the braces how you like them, just be consistent about it. 
And don't get confused by suggestion and issue.

Reply:
Alright cool